tools/sgen/input/FluentUIStyle.yml

```diff
@@ -285,9 +288,13 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
   textFont: $Typography.headline
   topPadding: $Spacing.medium
 
-AP_MSFDrawerTokens:
+AP_DrawerTokens:
```

Comment:
The Dark Mode scenario doesn't look correct. It seems like the List background should match on the elevation color as the drawer is doing, correct?
Can you work with @sophialee0416 to get the Left Nav scenario in sync with the design specs?

![drawer_leftnav_dark](https://user-images.githubusercontent.com/68076145/111218130-91ef5500-8593-11eb-9e6f-5e68e37bef26.png)

Reply:
This is now updated. 

![Screen Shot 2021-03-16 at 10 51 52 AM](https://user-images.githubusercontent.com/63682282/111358582-b99ff500-8647-11eb-9c69-5a93f6fc6c44.png)